src/braket/circuits/gates.py

```diff
@@ -454,11 +455,13 @@ class Rx(AngledGate):
     """X-axis rotation gate.
 
     Args:
-        angle (float): angle in radians.
+        angle (Union[FreeParameter, float]): angle in radians.
     """
 
-    def __init__(self, angle: float):
-        super().__init__(angle=angle, qubit_count=None, ascii_symbols=["Rx({:.3g})".format(angle)])
+    def __init__(self, angle: Union[FreeParameter, float]):
+        super().__init__(
+            angle=angle, qubit_count=None, ascii_symbols=["Rx({:})".format(str(angle))]
```

Comment:
Why get rid of formatting?

Reply:
Angle can be a string or a float. This is to accompany for the longer strings.